src/qml/ChatPage.qml

```diff
@@ -79,66 +75,108 @@ Kirigami.Page {
 			model: kaidan.messageModel
 
 			delegate: ChatMessage {
+				width: parent.width
 				sentByMe: model.recipient !== kaidan.jid
 				messageBody: model.message
 				dateTime: new Date(timestamp)
 				isRead: model.isDelivered
 				recipientAvatarUrl: kaidan.avatarStorage.getHashOfJid(author) !== "" ?
-						    kaidan.avatarStorage.getAvatarUrl(author) :
-						    kaidan.getResourcePath("images/fallback-avatar.svg")
+									kaidan.avatarStorage.getAvatarUrl(author) :
+									kaidan.getResourcePath("images/fallback-avatar.svg")
 			}
+			Controls.ScrollIndicator.vertical: Controls.ScrollIndicator {}
 		}
+	}
 
+	//
+	// Message Writing
+	//
+	footer: Controls.Pane {
+		id: sendingArea
+		layer.enabled: sendingArea.enabled
+		layer.effect: DropShadow {
+		  verticalOffset: 1
+		  color: Material.dropShadowColor
+		  samples: 20
+		  spread: 0.3
+		}
+		Layout.fillWidth: true
+		padding: 0
+		leftPadding: 12
+		wheelEnabled: true
+		background: Rectangle {
+			color: "white"
+		}
 
-		//
-		// Message Writing
-		//
-
-		Controls.Pane {
-			id: sendingArea
-			Layout.fillWidth: true
-			topPadding: Kirigami.Units.gridUnit * 0.1
-			bottomPadding: Kirigami.Units.gridUnit * 0.1
-			wheelEnabled: true
-			background: Rectangle {
-				color: "white"
-			}
-
-			RowLayout {
-				width: parent.width
-
-				Controls.TextField {
-					id: messageField
-					Layout.fillWidth: true
-					placeholderText: qsTr("Compose message")
-					wrapMode: Controls.TextArea.Wrap
-					selectByMouse: true
-					onAccepted: {
-						sendButton.onClicked()
+		RowLayout {
+			width: parent.width
+
+			/*Controls.ToolButton {
+				id: attachButton
+				Layout.preferredWidth: 60
+				Layout.preferredHeight: 60
+				Kirigami.Icon {
+					source: "document-send-symbolic"
+					isMask: true
+					smooth: true
+					anchors.centerIn: parent
+					width: 30
+					height: width
+				}
+			}*/
```

Comment:
Don't commit commented code.

Reply:
But actually I think, we can just include the attach button since it'll be just a dev build and I'm planning to add media sending to the next version.

Reply:
I committed it with that in mind.

Reply:
Then don't comment it out.